### ISSUE TITLE
Document the code

### ISSUE THREAD

**Issue by grdddj** [grdddj/similar-ordinals#14]
Each file and function should preferably have at least a short description

**Comment by Arsalaan-Alam** [comment]
Is this really needed? We can probably do it after we complete the MVP

**Comment by grdddj** [comment]
> Is this really needed? We can probably do it after we complete the MVP

Yeah, the priority is not high currently. OTOH, documenting the current state would be quite quick, because there is not a lot of code
